examples/api/java/FloatingPointArith.java

```diff
@@ -20,6 +20,8 @@
 
 import static io.github.cvc5.Kind.*;
 
+import java.math.RoundingMode;
+
```

Comment:
```suggestion
```

This should be `io.github.cvc5.RoundingMode`, which is already covered by the star import below.